pkg/util/crypto.go

```diff
@@ -97,3 +98,67 @@ func GenerateSelfSignedCert(host, certPath, keyPath string) error {
 
 	return nil
 }
+
+// CertPoolFromFile returns an x509.CertPool containing the certificates in the given PEM-encoded file.
+// Returns an error if the file could not be read, a certificate could not be parsed, or if the file does not contain any certificates
+func CertPoolFromFile(filename string) (*x509.CertPool, error) {
+	pool := x509.NewCertPool()
+
+	certs, err := CertificatesFromFile(filename)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, cert := range certs {
+		pool.AddCert(cert)
+	}
+
+	return pool, nil
+}
+
+// CertificatesFromFile returns the x509.Certificates contained in the given PEM-encoded file.
+// Returns an error if the file could not be read, a certificate could not be parsed, or if the file does not contain any certificates
+func CertificatesFromFile(file string) ([]*x509.Certificate, error) {
+	if len(file) == 0 {
```

Comment:
If no file is specified, I would expect you to return an error, since the function doc says that there will be an error if the file cannot be read (and you can't read a non-existent file).